README.md

```diff
@@ -17,7 +17,6 @@ The typical blockchain node runs multiple services in a single process. For exam
 Obscuro uses Trusted Execution Environments (TEE), like Intel SGX, to execute transactions in a confidential environment, which means we diverge from the typical architecture. 
```

Comment:
can't comment on the diagram. so will add here

On the diagram, can we represent somehow which connections are encrypted and which not. 
that should be very useful for readers.
For example on the rpc from wallet to we- unencrypted, and from we-host - encrypted

Reply:
Fixed.